expression/builtin_string_test.go

```diff
@@ -2447,3 +2447,45 @@ func (s *testEvaluatorSerialSuites) TestCIWeightString(c *C) {
 	checkResult("utf8mb4_general_ci", generalTests)
 	checkResult("utf8mb4_unicode_ci", unicodeTests)
 }
+
+func (s *testEvaluatorSuite) TestTranslate(c *C) {
+	cases := []struct {
+		args  []interface{}
+		isNil bool
+		isErr bool
+		res   string
+	}{
+		{[]interface{}{"ABC", "A", "B"}, false, false, "BBC"},
```

Comment:
include some test cases for binary input too :)

(for input think like `[]interface{[]byte{255}, []byte{255}, []byte{255}}` works. the output has to distinguish between bytes and strings)